src/NativeScript/ObjC/Inheritance/ObjCClassBuilder.mm

```diff
@@ -66,7 +67,7 @@ static void attachDerivedMachinery(GlobalObject* globalObject, Class newKlass, J
         ObjCSuperObject* superObject = ObjCSuperObject::create(vm, superStructure, derivedWrapper, globalObject);
         derivedWrapper->putDirect(vm, vm.propertyNames->superKeyword, superObject, ReadOnly | DontEnum | DontDelete);
 
-        [TNSValueWrapper attachValue:derivedWrapper toHost:instance];
+        globalObject->interop()->objectMap().set(instance, derivedWrapper);
```

Comment:
Oh, I've missed that. In that case would it be reasonable the constructor to add the object to the cache?